ChangeLog

```diff
@@ -1,5 +1,38 @@
 ### 3.9.1dev <- NOTE: The release version number will be 3.10.0 ###
 
+- Build: Updated bundled Qt6 to version 6.4.0 (#2892).
```

Comment:
I believe only the latest version should be in the changelog

Reply:
OK, will edit. I just used the `tools/changelog-helper.sh` script, and it generated these.